mods/ra/rules/fakes.yaml

```diff
@@ -39,8 +39,8 @@ WEAF:
 	Bib:
 	RenderBuilding:
 		Image: WEAP
-	WithIdleOverlay@ROOF:
-		Sequence: idle-top
+	WithProductionDoorOverlay:
```

Comment:
You could invent a new sequence `idle-door` with just one frame of the closed door.